kgcPy/__init__.py

```diff
@@ -4,6 +4,16 @@
 import io
 Image.MAX_IMAGE_PIXELS = None
 
+# Load the image file
+with resources.open_binary('kgcPy', 'kmz_int_reshape.png') as fp:
+    img = Image.open(fp)
+    img.load()
+
+with resources.open_binary('kgcPy', 'kg_zoneNum.csv') as fp:
+    df = pd.read_csv(fp)
```

Comment:
`df` and `img` are used as locals inside other functions in this module; I think there shouldn't be any scope issue, but just to be safe it might make sense to call the module-level vars something else.